cdci_data_analysis/flask_app/templates/email.html

```diff
@@ -57,8 +57,9 @@
 To learn how to get a new token, please refer to <a href="{{ oda_site.manual_reference }}">these instructions</a>.<br>
 <br>
 {% else %}
-Please note that we were not able to embed API code in this email, since exceptionally, it was too large. 
+Please note that we were not able to embed API code in this email, since exceptionally, it was too large.
 Our support team has been alerted about this incident.
+However, a python script with the api code has been attached to this email.
```

Comment:
It's not really that problematic anymore. Just less favorable. Also I do not think we get an alert?

```suggestion
Please note the API code for this query was too large to embed it in the email text. Instead, we attach it as a python script.
```